luigi/worker.py

```diff
@@ -159,13 +159,15 @@ def run(self):
             self.task.trigger_event(Event.FAILURE, self.task, ex)
             subject = "Luigi: %s FAILED" % self.task
 
-            error_message = notifications.wrap_traceback(self.task.on_failure(ex))
+            raw_error_message = self.task.on_failure(ex)
+            notification_error_message = notifications.wrap_traceback(raw_error_message)
+            expl = json.dumps(raw_error_message)
```

Comment:
I don't  know this code really. But didn't you add ` json.dumps()` call here? So it's not just refactoring here right?